contracts/.solhint.json

```diff
@@ -9,13 +9,21 @@
     "gas-increment-by-one": "error",
     "gas-struct-packing": "error",
     "no-unused-vars": "error",
+    "no-unused-import": "error",
     "interface-starts-with-i": "error",
-    "reason-string": "warn",
-    "private-vars-leading-underscore": "warn",
-    "gas-length-in-loops": "warn",
-    "gas-strict-inequalities": "warn",
-    "ordering": "warn",
-    "gas-indexed-events": "warn",
-    "gas-custom-errors": "warn"
+    "ordering": "error",
+    "private-vars-leading-underscore": "error",
+    "gas-length-in-loops": "error",
+    "const-name-snakecase": "error",
+    "contract-name-camelcase": "error",
+    "event-name-camelcase": "error",
+    "func-name-mixedcase": "error",
+    "immutable-vars-naming": "error",
+    "use-forbidden-name": "error",
+    "var-name-mixedcase": "error",
+    "imports-on-top": "error",
+    "visibility-modifier-order": "error",
+    "reason-string": "error",
+    "gas-custom-errors": "off"
```

Comment:
@Mikelle I turned this one off due to https://github.com/ethereum/go-ethereum/issues/26823